docker-compose.override.yml.dist

```diff
@@ -10,9 +10,17 @@ services:
       - '443:443'
     environment:
       XDEBUG_REMOTE_ENABLED: 'false'
-      TIDEWAYS_ENABLED: 'true'
+      XDEBUG_REMOTE_HOST: '10.254.254.254'
```

Comment:
docker for mac has a dns entry now for the host `docker.for.mac.localhost`, and doesn't need a custom launchd script that adds this ip.

Reply:
as for linux, I believe it can use the gateway ip, windows docker.for.win.localhost

Reply:
I'll leave this for now as it's a complex fix and not that often everyone uses xdebug remote debugging now that they can't just follow the readme